src/Symfony/ParameterMap.php

```diff
@@ -18,6 +18,6 @@ public function getParameter(string $key): ?ParameterDefinition;
 	/**
 	 * @return array<string>
 	 */
-	public static function getParameterKeysFromNode(Expr $node, Scope $scope): array;
+	public function getParameterKeysFromNode(Expr $node, Scope $scope): array;
```

Comment:
This might be considered a breaking change and thus require a major version bump?